specs/proposals/blob-signing.md

```diff
@@ -0,0 +1,282 @@
+
+    Below is an example of the manifest for the reference `myregistry/mypath/mysboms:v1`:
+
+    ```json  
+    {
+        "schemaVersion": 2,
+        "mediaType": "application/vnd.oci.image.manifest.v1+json",
+        "artifactType": "application/spdx+json",
+        "config": {
+            "mediaType": "application/vnd.oci.empty.v1+json",
+            "digest": "sha256:44136fa355b3678a1146ad16f7e8649e94fb4fc21fe77e8310c060f61caaff8a",
+            "size": 2,
+            "data": "e30="
+        },
+        "layers": [
+            {
+            "mediaType": "application/json",
+            "digest": "sha256:3154705eb5fc92bc3f91deec6008e6dd4a02b3e53ac18bdb7ea3ff03e62971a8",
+            "size": 845,
+            "annotations": {
+                "org.opencontainers.image.title": "sbom.json"
+            }
+            }
+        ],
+        "annotations": {
+            "org.opencontainers.image.created": "2025-02-18T06:00:00Z"
+        }
+    }
+    ```
+
+    Below is an example of the manifest for the blob signature in the registry:
+
+    ```json
+    {
+        "schemaVersion": 2,
+        "mediaType": "application/vnd.oci.image.manifest.v1+json",
+        "artifactType": "application/vnd.cncf.notary.signature",
+        "config": {
+            "mediaType": "application/vnd.oci.empty.v1+json",
+            "digest": "sha256:44136fa355b3678a1146ad16f7e8649e94fb4fc21fe77e8310c060f61caaff8a",
+            "size": 2,
+            "data": "e30="
+        },
+        "layers": [
+            {
+            "mediaType": "application/cose",
+            "digest": "sha256:6dc2a336b124d625f87deb3f539fa1465414ddad300a898adf4682e530d6f144",
+            "size": 1479,
+            "annotations": {
+                "org.opencontainers.image.title": "sbom.json.cose.sig"
+            }
+            }
+        ],
+        "subject": {
+            "mediaType": "application/vnd.oci.image.manifest.v1+json",
+            "digest": "sha256:57eed31a09556933bddd4e164e1e085a220cf00489477eb98d89feac902eb42b",
+            "size": 553
+        },
+        "annotations": {
+            "io.cncf.notary.blob.signature": "true",
+            "io.cncf.notary.x509chain.thumbprint#S256": "[\"aaaa\", \"bbbb\"]",
+            "org.opencontainers.image.created": "2025-02-18T06:08:45Z"
+        }
+    }
+    ```
+
+    Alternatively, you can also use ORAS tool to push the blob file first, and then use Notation CLI command to push only the signature:
+
+    ```shell
+    oras push myregistry/mypath/mysboms:v1 sbom.json:application/octet-stream 
+    notation blob push --signature sbom.json.cose.sig --reference myregistry/mypath/mysboms:v1
+    ```
+
+    You can also create an oci image layout and then use ORAS tool to copy it to the target registry. 
```

Comment:
Why does it need to be OCI layout?

Reply:
This is another opition. That users can create OCI layout for signature first and then copy to the registry.

Reply:
Remove OCI image layout from the proposal. It will be brought back if there is a clear scenario.